code/game/objects/structures/bookcase.dm

```diff
@@ -26,6 +45,10 @@
 		else
 			name = ("bookcase ([strip_html(newname)])")
 			playsound(src, "paper_writing", 15, TRUE)
+	else if(HAS_TRAIT(O, TRAIT_TOOL_WRENCH))
+		playsound(src.loc, 'sound/items/Ratchet.ogg', 25, 1)
+		if(do_after(user, 1 SECONDS, INTERRUPT_MOVED, BUSY_ICON_FRIENDLY, src))
+			deconstruct(FALSE)
```

Comment:
shouldn't that be TRUE as it's disassembled proper?

Reply:
doesn't really matter but sure